lib/logcat.js

```diff
@@ -13,13 +13,26 @@ class Logcat {
   }
 
   startCapture () {
-    return new Promise(async (resolve, reject) => {
-      log.debug("Starting logcat capture");
+    let started = false;
+    return new Promise(async (_resolve, _reject) => {
+      const resolve = function (...args) {
+        started = true;
+        _resolve(...args);
+      };
+      const reject = function (...args) {
+        started = true;
+        _reject(...args);
+      };
+
+      log.debug('Starting logcat capture');
       this.proc = new SubProcess(this.adb.path, this.adb.defaultArgs.concat(['logcat', '-v', 'threadtime']));
-      await this.proc.start(0);
       this.proc.on('exit', (code, signal) => {
         log.error(`Logcat terminated with code ${code}, signal ${signal}`);
         this.proc = null;
+        if (!started) {
+          log.warn('Logcat not started. Continuing');
+          resolve();
```

Comment:
They could be removed, but then the setting of `started` would have to be all over the place.